api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/FSGEnrollmentDataConverter.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.ugandaemrreports.data.converter;
+
+import org.openmrs.Concept;
+import org.openmrs.Obs;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+
+/**
+ */
+public class FSGEnrollmentDataConverter implements DataConverter {
+    @Override
+    public Object convert(Object obj) {
```

Comment:
@solemabrothers Why return an empty string in some cases and null in others, be consistent